### PR TITLE
Print message on sucessfull copy of image

### LINKED ISSUE #1530: Fix console message when a image is copied successfully.
On LXD 0.27, when a image is copied successfully the percentage remains on the new line. Why not add a proper message like "image copied successfully".

```
ishwar@lxd-host:~$ lxc image copy images:debian/jessie/amd64 local: --alias jessie
ishwar@lxd-host:~$ 100%
```

### PR DESCRIPTION
**Pull request by nehaljwani**
Example:
[root@lxc-dev ~]# lxc image copy images:centos/6/i386 local: --alias c6
Image copied succesfully!
[root@lxc-dev ~]#

Fixes #1530

Signed-off-by: Nehal J Wani <nehaljw.kkd1@gmail.com>

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -644,7 +644,7 @@ func (c *Client) CopyImage(image string, dest *Client, copy_aliases bool, aliase
 		}
 	}
 
-	return nil
+	return err
```
Comment (tych0) [inline client.go]: What's the deal with this hunk? It looks wrong.

Comment (tych0) [inline client.go]: Oh, I see. Should we perhaps fail on the first alias here?

**Comment by tych0** [comment]
jenkins: test this please

**Comment by stgraber** [comment]
jenkins: test this please

**Comment by tych0** [comment]
Amazing. I think github is eating my "jenkins: test this please" comments.

**Comment by stgraber** [comment]
Needs a run of "make i18n"

**Comment by nehaljwani** [comment]
I have to send the updated po/lxd.pot along with the patch?

**Comment by stgraber** [comment]
Right, po/lxd.pot is in the git branch and so must be updated every time a string is added, removed or changed in one of the translated command. "make i18n" will do that for you and you should indeed include the resulting code change in your commit.

**Comment by nehaljwani** [comment]
po/lxd.pot added. Looks like we need to update this info at CONTRIBUTING.md :)

File: lxc/image.go [hunk 2]
```diff
@@ -210,7 +210,11 @@ func (c *imageCmd) run(config *lxd.Config, args []string) error {
 			fmt.Printf(i18n.G("Copying the image: %s")+"\r", progress)
 		}
 
-		return d.CopyImage(image, dest, copyAliases, addAliases, publicImage, progressHandler)
+		err = d.CopyImage(image, dest, copyAliases, addAliases, publicImage, progressHandler)
+		if err == nil {
+			fmt.Println(i18n.G("Image copied sucessfully!"))
```
Comment (brauner) [inline lxc/image.go]: Chiming in here. sorry for that, but this should be "successfully". Spelling mistakes look rather unprofessional. :)

File: po/lxd.pot [hunk 3]
```diff
@@ -346,7 +346,11 @@ msgstr  ""
 msgid   "If this is your first run, you will need to import images using the 'lxd-images' script."
 msgstr  ""
 
-#: lxc/image.go:335
+#: lxc/image.go:215
+msgid   "Image copied sucessfully!"
```
Comment (brauner) [inline po/lxd.pot]: Same as above. :): Chiming in here. sorry for that, but this should be "successfully". Spelling mistakes look rather unprofessional. :)

File: lxc/image.go (hunk 2 above)
Comment (nehaljwani) [inline lxc/image.go]: Oops. My bad. :'( 
https://github.com/lxc/lxd/pull/1539

**Pull request merged**